flint.ui/src/store/modules/rothc.js

```diff
@@ -392,28 +400,97 @@ export default {
+      let preprocessed_RothC_string = final_RothC_string.replaceAll('"$#', '')
       let final_RothC_config_string = preprocessed_RothC_string.replaceAll(
-        '#$"',
-        ' '
+        '$#"',
+        ''
       )
+
+      console.log(final_RothC_config_string)
       axios
         .post('http://127.0.0.1:8080/rothc', final_RothC_config_string)
         .then((response) => {
           this._vm.$toast.success(`${response}`, { timeout: 2000 })
           console.log(response)
+          //this.state.received_data = response.data;
+          commit('set_received_rothc_example_data', response.data)
+          console.log(this.state.rothc.received_rothc_example_data)
         })
         .catch((error) => {
           this._vm.$toast.error(`${error}`, { timeout: 2000 })
           console.log(error)
         })
+    },
+    process_RothC_config({ commit }) {
+      console.log(this.state.rothc.received_rothc_example_data)
+
+      const dataForge = require('data-forge')
+
+      console.log('running from state')
+      console.log(this.state.rothc.received_rothc_example_data)
+      var dataset = this.state.rothc.received_rothc_example_data
+      var RothC_module_DPM = [],
+        RothC_module_RPM = [],
+        RothC_module_BIO = [],
+        RothC_module_HUM = [],
+        RothC_module_IOM = [],
+        simulation_step = []
+
+      let lines = (dataset || '').split('\n')
+      lines.splice(0, 4)
+      lines.splice(-4)
+      dataset = lines.join('\n')
+      const df = dataForge.fromCSV(dataset)
+      let df_as_array = df.toArray()
+      console.log(typeof df_as_array)
+      console.log('array1')
+      // console.log(df_as_array[0]['Pool 1'])
+      console.log(df_as_array.length)
+
+      for (let step = 0; step < df_as_array.length; step++) {
+        pool_1[step] = parseFloat(df_as_array[step]['Pool 1'])
+        pool_2[step] = parseFloat(df_as_array[step]['Pool 2'])
+        pool_3[step] = parseFloat(df_as_array[step]['Pool 3'])
+        let x = step
+        simulation_step[step] = x
+      }
+
+      console.log('pool 1')
+      for (let step = 0; step < df_as_array.length; step++) {
+        console.log(pool_1[step])
+      }
+      console.log(simulation_step)
+      console.log(pool_1)
+      console.log(pool_2)
+      console.log(pool_3)
+
+      this.state.rothc.Point_config_pool_1 = pool_1
+      this.state.rothc.Point_config_pool_2 = pool_2
+      this.state.rothc.Point_config_pool_3 = pool_3
+
+      commit('update_Point_config_pool_1', pool_1)
+      commit('update_Point_config_pool_2', pool_2)
+      commit('update_Point_config_pool_3', pool_3)
+
+      console.log('this.Point_config_pool_1')
+      console.log(this.state.rothc.Point_config_pool_1)
+      console.log('this.Point_config_pool_2')
+      console.log(this.state.rothc.Point_config_pool_2)
+      console.log('this.Point_config_pool_3')
+      console.log(this.state.rothc.Point_config_pool_3)
```

Comment:
This part looks like it needs to be specialised for RothC? Maybe still in progress! 🚀